### DISCUSSION TITLE
Github puanları

### DISCUSSION THREAD

**Discussion by King71756** [IBAX-io/go-ibax#494] [Q&A]
Github puanları nasıl alınıyor bilgisi olan var mı ?

**Comment by mackcarelock** [comment]
Aynen 🙋🏻‍♂️

**Comment by ibaxmustafaceviz** [comment]
bende merak ediyorum. bilen biri varsa cevaplayabilir mi?

**Comment by hsynkader** [comment]
3000 puandan bahsediyorsanız, ana sayfada github bağlantısına tıkladıkdan sonra çıkan görsel üzerinde star ve fork var. Onlara ayrı ayrı tıklayın. Github a yönlendirileceksiniz. Github da sağ üst köşede star ve fork u tıklayın İBAX ana sayfada twitter da rt yaparken çıkan ekran gibi finishing ibaresini tıklamanız yeterli.
